contracts/base/ModuleManager.sol

```diff
@@ -289,7 +305,7 @@ abstract contract ModuleManager is Storage, Receiver, EIP712, IModuleManagerEven
         // If a validator module is uninstalled and reinstalled without proper authorization, it can compromise
         // the account's security and integrity. By restricting these selectors, we ensure that the fallback handler
         // cannot be manipulated to disrupt the expected behavior and security of the account.
-        require(!(selector == bytes4(0x6d61fe70) || selector == bytes4(0x8a91b0e3)), FallbackSelectorForbidden());
+        require(!(selector == bytes4(0x6d61fe70) || selector == bytes4(0x8a91b0e3) || selector == bytes4(0)), FallbackSelectorForbidden());
```

Comment:
add a comment that you don't support fallback handlers for empty bytes (`receive`) and explicit `bytes4(0)` selectors?

Reply:
done